### DISCUSSION TITLE
custom providers

### DISCUSSION THREAD

**Discussion by kesar** [wevm/wagmi#349] [Q&A] [answered]
has anyone implemented other providers? I was looking some integration w torus wallet / portis / magic link to have social login in our site.

**Comment by tmm** [comment, marked as answer]
PRs open for MM (https://github.com/tmm/wagmi/pull/279) and Gnosis (https://github.com/tmm/wagmi/pull/336) connectors.

**Comment by hsuanweifu-sports-inference** [comment]
Seem like @kesar has made a Magic Link connector here here https://github.com/EveripediaNetwork/wagmi-magic-connector. I'd still like to see it being included in the wagmi projects though. Maybe as an optional dependency.

I wasn't aware of this resource and made my own magic connector and was in the process of making a pr into wagmi.
